bin/Powheg/runcmsgrid_powheg.sh

```diff
@@ -97,6 +97,13 @@ export PYTHONPATH=.:${PYTHONPATH}
 #ln -s `which gfortran` g77
 export PATH=`pwd`:${PATH}
 
+if [ "${process}" == "X0jj" ]; then
+    cp -p ${WORKDIR}/MadLoopParams.dat .
+    for f in `ls ${WORKDIR}/MG5_aMC_v2_6_7/X0jj/SubProcesses/MadLoop5_resources/*`
```

Comment:
should we make the path such that it is independent from the exact version so that it is easier to change when switching?